app/graph/relay_on_rails_schema.rb

```diff
@@ -13,7 +13,7 @@ class RelayOnRailsSchema < GraphQL::Schema
 
   lazy_resolve(Concurrent::Future, :value)
 
-  disable_introspection_entry_points
+  disable_introspection_entry_points unless Rails.env.development?
```

Comment:
FYI @jayjay-w , I had to add this so we're able to update the Relay schema file.